### ISSUE TITLE
Bug: Cannot run any command.

### ISSUE THREAD

**Issue by ricky16x** [nathom/streamrip#105]
## Bug Report

**Current Behavior**
Even if I just use ```rip``` or ```rip -help```, it throws error.

**Command used:** `rip`

```C:\Windows\system32>rip
Traceback (most recent call last):
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Rounak\AppData\Local\Programs\Python\Python39\Scripts\rip.exe\__main__.py", line 7, in <module>
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\streamrip\cli.py", line 415, in main
    cli(obj={})
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1646, in invoke
    super().invoke(ctx)
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\click\decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\streamrip\cli.py", line 69, in cli
    config = Config()
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\streamrip\config.py", line 57, in __init__
    self.load()
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\streamrip\config.py", line 92, in load
    for k, v in tomlkit.loads(cfg.read().strip()).items():
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\tomlkit\api.py", line 34, in loads
    return parse(string)
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\tomlkit\api.py", line 51, in parse
    return Parser(string).parse()
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\tomlkit\parser.py", line 160, in parse
    key, value = self._parse_table()
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\tomlkit\parser.py", line 1154, in _parse_table
    item = self._parse_item()
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\tomlkit\parser.py", line 306, in _parse_item
    return self._parse_key_value(True)
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\tomlkit\parser.py", line 397, in _parse_key_value
    val = self._parse_value()
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\tomlkit\parser.py", line 561, in _parse_value
    return self._parse_basic_string()
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\tomlkit\parser.py", line 854, in _parse_basic_string
    return self._parse_string(StringType.SLB)
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\tomlkit\parser.py", line 990, in _parse_string
    value += self._parse_escaped_char(delim.is_multiline())
  File "c:\users\rounak\appdata\local\programs\python\python39\lib\site-packages\tomlkit\parser.py", line 897, in _parse_escaped_char
    raise self.parse_error(InvalidUnicodeValueError)
tomlkit.exceptions.InvalidUnicodeValueError: Invalid unicode value at line 3 col 13

**Comment by nathom** [comment]
Please include a copy of your redacted config file. Thanks.

**Comment by ricky16x** [comment]
How do I do that?
`rip config -o` throws the above error.

**Comment by nathom** [comment]
Check the `\Users\<username>\AppData\Local\streamrip` or `C:\Users\<username>\AppData\Roaming\streamrip` folders for a `config.toml`.

**Comment by ricky16x** [comment]
`[downloads]
# Folder where tracks are downloaded to
folder = "C:\Users\Rounak\Music"
# Put Qobuz albums in a 'Qobuz' folder, Tidal albums in 'Tidal' etc.
source_subdirectories = false
# Download (and convert) tracks all at once, instead of sequentially. 
# If you are converting the tracks, or have fast internet, this will 
# substantially improve processing speed.
concurrent = false

[qobuz]
# 1: 320kbps MP3, 2: 16/44.1, 3: 24/<=96, 4: 24/>=96
quality = 3
# This will download booklet pdfs that are included with some albums
download_booklets = true

email = ""
# This is an md5 hash of the plaintext password
password = ""
# Do not change
app_id = ""
# Do not change
secrets = []

[tidal]
# 0: 256kbps AAC, 1: 320kbps AAC, 2: 16/44.1 "HiFi" FLAC, 3: 24/44.1 "MQA" FLAC
quality = 3
# This will download videos included in Video Albums.
download_videos = true

# Do not change any of the fields below
user_id = ""
country_code = ""
access_token = ""
refresh_token = ""
# Tokens last 1 week after refresh. This is the Unix timestamp of the expiration
# time. If you haven't used streamrip in more than a week, you may have to log
# in again using `rip config --tidal`
token_expiry = ""

# Doesn't require login
[deezer]
# 0, 1, or 2
quality = 2

[soundcloud]
# Only 0 is available for now
quality = 0

[youtube]
# Only 0 is available for now
quality = 0
# Download the video along with the audio
download_videos = false
# The path to download the videos to
video_downloads_folder = ""

# This stores a list of item IDs so that repeats are not downloaded.
[database]
enabled = true
path = "C:\\Users\\Rounak\\AppData\\Roaming\\streamrip\\downloads.db"

# Convert tracks to a codec after downloading them.
[conversion]
enabled = false
# FLAC, ALAC, OPUS, MP3, VORBIS, or AAC
codec = "ALAC"
# In Hz. Tracks are downsampled if their sampling rate is greater than this. 
# Value of 48000 is recommended to maximize quality and minimize space
sampling_rate = 48000
# Only 16 and 24 are available. It is only applied when the bit depth is higher
# than this value.
bit_depth = 24

# Filter a Qobuz artist's discography. Set to 'true' to turn on a filter.
[filters]
# Remove Collectors Editions, live recordings, etc.
extras = false
# Picks the highest quality out of albums with identical titles.
repeats = false
# Remove EPs and Singles
non_albums = false
# Remove albums whose artist is not the one requested
features = false
# Skip non studio albums
non_studio_albums = false
# Only download remastered albums
non_remaster = false

[artwork]
# Write the image to the audio file
embed = true
# The size of the artwork to embed. Options: thumbnail, small, large, original.
# "original" images can be up to 30MB, and may fail embedding. 
# Using "large" is recommended.
size = "large"
# Save the cover image at the highest quality as a seperate jpg file
keep_hires_cover = true

[metadata]
# Sets the value of the 'ALBUM' field in the metadata to the playlist's name. 
# This is useful if your music library software organizes tracks based on album name.
set_playlist_to_album = true
# Replaces the original track's tracknumber with it's position in the playlist
new_playlist_tracknumbers = true

# Changes the folder and file names generated by streamrip.
[filepaths]
# Create folders for single tracks within the downloads directory using the folder_format
# template
add_singles_to_folder = false
# Available keys: "albumartist", "title", "year", "bit_depth", "sampling_rate",
# and "container"
folder_format = "{albumartist} - {title} ({year}) [{container}] [{bit_depth}B-{sampling_rate}kHz]"
# Available keys: "tracknumber", "artist", "albumartist", "composer", and "title"
track_format = "{tracknumber}. {artist} - {title}"

# Last.fm playlists are downloaded by searching for the titles of the tracks
[lastfm]
# The source on which to search for the tracks.
source = "qobuz"
# If no results were found with the primary source, the item is searched for 
# on this one.
fallback_source = "deezer"

[misc]
# Check whether a newer version of streamrip is available when starting up
check_for_updates = true

# Metadata to identify this config file. Do not change.
version = "0.6.1"`

**Comment by nathom** [comment]
You need to escape backslashes (`\`) by replacing them with `\\` in `downloads.folder`. See the path here as an example

```toml
[database]
enabled = true
path = "C:\\Users\\Rounak\\AppData\\Roaming\\streamrip\\downloads.db"
```

This will be fixed in the next update so that you don't have to manually input the downloads directory.

**Comment by ricky16x** [comment]
Hmm worked, also is there a way to remove a song from the 'db'? So i can download it again.

**Comment by Gamer4Life2020** [comment]
> Hmm worked, also is there a way to remove a song from the 'db'? So i can download it again.

Add -nd or --no-db to the end of whatever you want to DL for no database or delete the downloads.db file, same place as the config. 

 https://play.qobuz.com/artist/1234 -nd

**Comment by nathom** [comment]
Closing because the issue was fixed in `v0.6.4`

**Comment by zyndin** [comment]
Hello,

May I ask how to fix with this issue

`PS C:\streamrip> rip config --tidal

  InvalidUnicodeValueError

  Invalid unicode value at line 3 col 13

  at c:\users\vyzen\appdata\local\programs\python\python39\lib\site-packages\tomlkit\parser.py:897 in _parse_escaped_char
       893│                 self.inc_n(len(ue) + 1)
       894│
       895│                 return u
       896│
    →  897│             raise self.parse_error(InvalidUnicodeValueError)
       898│
       899│         raise self.parse_error(InvalidCharInStringError, self._current)
       900│
       901│     def _parse_string(self, delim):  # type: (StringType) -> String

If this was unexpected, please open a Bug Report at https://github.com/nathom/streamrip/issues/new/choose
PS C:\streamrip>`
